### ISSUE TITLE
[GraphQl] Custom giftcard amount is not returned in the response if request doesn't have "allow_open_amount" set

### ISSUE THREAD

**Issue by magento-engcom-team** [magento/magento2#30851]
####  

```java

addRequisitionListItemsToCart(
        requisitionListUid: ID, @doc(description: "The unique ID of the requisition list")
        requisitionListItemUids: <ID!> @doc(description: "An array of UIDs presenting products to be added to the cart. If no UIDs are specified, all items in the requisition list will be added to the cart")
    ): AddRequisitionListItemsToCartOutput @resolver(class: "Magento\\RequisitionListGraphQl\\Model\\Resolver\\RequisitionList\\AddToCart") @doc(description: "Add items in the requisition list to the customer's cart")

```

**Comment by magento-engcom-team** [comment]
The issue was exported from the internal JIRA. The link to the original JIRA issue: https://jira.corp.magento.com/browse/MC-38979

**Comment by magento-engcom-team** [comment]
Hi @magento-engcom-team, @sasha19957099. 

Thank you for your report and collaboration!

The related internal Jira ticket `MC-38979` was closed as `Fixed`. 

The fix will be available with the upcoming `2.4.3` release.
